deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx

```diff
@@ -43,33 +45,37 @@ export const Tooltip = (props: TooltipProps) => {
               'radix-side-right:animate-slide-left-fade',
               'radix-side-bottom:animate-slide-up-fade',
               'radix-side-left:animate-slide-right-fade',
-              'inline-flex items-center rounded-lg px-3 py-2 shadow-sm',
-              'bg-gray-900 dark:bg-gray-700 max-w-xs',
+              'rounded-md pl-3 py-2 shadow-sm w-[200px]',
```

Comment:
let's not fix width, we can do max width.

Reply:
added max width